features/enlinkd/service/impl/src/main/java/org/opennms/netmgt/enlinkd/service/impl/LldpTopologyServiceImpl.java

```diff
@@ -214,8 +223,17 @@ public void setLldpElementDao(LldpElementDao lldpElementDao) {
                     parsed.add(sourceLink.getId());
                     continue;
                 }
+                String sourceLldpSysname = nodelldpelementidMap.get(sourceLink.getNodeId()).getLldpSysname();
+                if (sourceLldpSysname.equals(sourceLink.getLldpRemSysname())) {
+                    if (LOG.isInfoEnabled()) {
+                        LOG.info("getLldpLinks: self link not adding source: {}",sourceLink);
+                    }
```

Comment:
Just a note: there is no need in guarding log messages like this. Even the formatting will be done only if the log level is enabled. This is only required if there is some computation related to the log message.

Reply:
Ok.

Reply:
those are the classes that use this:

./adapters/collectors/bridge/src/main/java/org/opennms/netmgt/enlinkd/DiscoveryBridgeDomains.java:                if (LOG.isDebugEnabled()) {
./adapters/collectors/bridge/src/main/java/org/opennms/netmgt/enlinkd/DiscoveryBridgeDomains.java:            if (LOG.isDebugEnabled()) {
./adapters/collectors/ipnettomedia/src/main/java/org/opennms/netmgt/enlinkd/NodeDiscoveryIpNetToMedia.java:                if (LOG.isDebugEnabled()) {
./adapters/updaters/bridge/src/main/java/org/opennms/netmgt/enlinkd/BridgeOnmsTopologyUpdater.java:            if (LOG.isDebugEnabled()) {
./service/impl/src/main/java/org/opennms/netmgt/enlinkd/service/impl/CdpTopologyServiceImpl.java:            if (LOG.isDebugEnabled()) {
./service/impl/src/main/java/org/opennms/netmgt/enlinkd/service/impl/CdpTopologyServiceImpl.java:            if (LOG.isDebugEnabled()) {
./service/impl/src/main/java/org/opennms/netmgt/enlinkd/service/impl/BridgeTopologyServiceImpl.java:        if (LOG.isDebugEnabled()) {
./service/impl/src/main/java/org/opennms/netmgt/enlinkd/service/impl/BridgeTopologyServiceImpl.java:        if (LOG.isDebugEnabled()) {
./service/impl/src/main/java/org/opennms/netmgt/enlinkd/service/impl/BridgeTopologyServiceImpl.java:                    if (LOG.isDebugEnabled()) {
./service/impl/src/main/java/org/opennms/netmgt/enlinkd/service/impl/BridgeTopologyServiceImpl.java:            if (LOG.isDebugEnabled()) {
./service/impl/src/main/java/org/opennms/netmgt/enlinkd/service/impl/IpNetToMediaTopologyServiceImpl.java:            if (LOG.isDebugEnabled()) {
./service/impl/src/main/java/org/opennms/netmgt/enlinkd/service/impl/IpNetToMediaTopologyServiceImpl.java:            if (LOG.isDebugEnabled()) {
./service/impl/src/main/java/org/opennms/netmgt/enlinkd/service/impl/LldpTopologyServiceImpl.java:                    if (LOG.isInfoEnabled()) {
./service/impl/src/main/java/org/opennms/netmgt/enlinkd/service/impl/LldpTopologyServiceImpl.java:                    if (LOG.isInfoEnabled()) {
./service/impl/src/main/java/org/opennms/netmgt/enlinkd/service/impl/LldpTopologyServiceImpl.java:                if (LOG.isInfoEnabled()) {
./service/impl/src/main/java/org/opennms/netmgt/enlinkd/service/impl/LldpTopologyServiceImpl.java:                    if (LOG.isInfoEnabled()) {
./service/impl/src/main/java/org/opennms/netmgt/enlinkd/service/impl/LldpTopologyServiceImpl.java:                    if (LOG.isInfoEnabled()) {
./service/impl/src/main/java/org/opennms/netmgt/enlinkd/service/impl/LldpTopologyServiceImpl.java:                    if (LOG.isInfoEnabled()) {
./service/impl/src/main/java/org/opennms/netmgt/enlinkd/service/impl/LldpTopologyServiceImpl.java:                if (LOG.isInfoEnabled()) {
./service/impl/src/main/java/org/opennms/netmgt/enlinkd/service/impl/OspfTopologyServiceImpl.java:            if (LOG.isDebugEnabled()) {
./service/impl/src/main/java/org/opennms/netmgt/enlinkd/service/impl/IsisTopologyServiceImpl.java:            if (LOG.isDebugEnabled()) {
./service/impl/src/main/java/org/opennms/netmgt/enlinkd/service/impl/IsisTopologyServiceImpl.java:            if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/BridgeSimpleConnection.java:        if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/BridgeSimpleConnection.java:        if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/BridgeSimpleConnection.java:        if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/BridgeSimpleConnection.java:                if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/BridgeSimpleConnection.java:                if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/BridgeSimpleConnection.java:            if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/BridgeSimpleConnection.java:        if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/BridgeSimpleConnection.java:                if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/BridgeSimpleConnection.java:                if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/DiscoveryBridgeTopology.java:            if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/DiscoveryBridgeTopology.java:            if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/DiscoveryBridgeTopology.java:        if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/DiscoveryBridgeTopology.java:        if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/DiscoveryBridgeTopology.java:            if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/DiscoveryBridgeTopology.java:                    if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/DiscoveryBridgeTopology.java:                    if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/DiscoveryBridgeTopology.java:                  if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/DiscoveryBridgeTopology.java:        if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/DiscoveryBridgeTopology.java:                if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/DiscoveryBridgeTopology.java:        if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/DiscoveryBridgeTopology.java:            if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/DiscoveryBridgeTopology.java:                if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/DiscoveryBridgeTopology.java:                if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/BridgeForwardingTable.java:            if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/BridgeForwardingTable.java:                if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/BridgeForwardingTable.java:                if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/BridgeForwardingTable.java:            if (LOG.isDebugEnabled()) {
./service/api/src/main/java/org/opennms/netmgt/enlinkd/service/api/BridgeForwardingTable.java:            if (LOG.isDebugEnabled()) {